src/components/HowItWorksSection.tsx

```diff
@@ -0,0 +1,96 @@
+import Image, { StaticImageData } from "next/image";
+import Step1Button from "../../public/step1-button.png";
+import Step2Describe from "../../public/step2-describe-chat.png";
+import Step3Answer from "../../public/step3-answer.png";
+
+interface StepCardProps {
+    stepNumber: number;
+    title: string;
+    description: string;
+    image: StaticImageData;
+    reverse?: boolean;
+    width?: number;
+    height?: number;
+}
+
+function StepCard({
+    stepNumber,
+    title,
+    description,
+    image,
+    reverse,
+    width = 300,
+    height = 300,
+}: StepCardProps) {
+    return (
+        <div
+            className={`flex flex-col md:flex-row items-center gap-8 ${reverse ? "md:flex-row-reverse" : ""}`}>
+            <div className="w-full md:w-1/2 px-8 text-xl">
+                <p className="text-colorMuted">Krok {stepNumber}</p>
+                <p className="text-3xl pt-2 font-semibold">{title}</p>
+                <p className="pt-3">{description}</p>
+            </div>
+            <div className="flex justify-center items-center w-1/2">
+                <Image src={image} alt={`Krok ${stepNumber}`} width={width} height={height} />
```

Comment:
That's definitely nitpicking by me, but I'm not sure if we need alt props for the images because they're kinda decorations and also there are descriptions next to them - however if we need alt text for them I'd consider changing it for more descriptive